### PR TITLE
Dance party landing page - tutorial section

### PR DESCRIPTION
**Pull request by nkiruka**
This PR is for the creation of the dance party landing page.  There will be multiple PRs for each section of the landing page.

### Implementation - first section of landing page that includes dance party boxes
- Added the boxes with text
- Image placeholder
- Adjusted box heading font size

Additional updates to this section will be required such as:
-Replace the current image
-Add correct links for "help translate page" and for each tutorial
-Any required styling

### Screen shot
screen shot 2018-11-03 at 4 34 41 am

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/css/dance-landing.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  right: 20px;
+}
+
+.dance-blocks-tutorial {
+  right: 270px;
+}
+
+.dance-container .tutorial-box {
+  background-color: rgba(0, 0, 0, 0.4);
+  color:white;
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/css/dance-landing.css]: tiny nit: space

File: pegasus/sites.v3/code.org/public/css/dance-landing.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+}
+
+.dance-tutorial-specs {
+  font-size: 12px;
+  line-height: 16px;
+}
+
+.dance-try-button {
+  font-size:18px;
+  height:40px;
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/css/dance-landing.css]: tiny nit: space line 77 & 78

File: pegasus/sites.v3/code.org/public/css/dance-landing.css [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+}
+
+.header-banner {
+  border-radius: 5px;
+  width: 100%
+}
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/css/dance-landing.css]: this element isn't in your haml? neither is img-background. Why have css for them?

File: pegasus/sites.v3/code.org/public/css/dance-landing.css [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+  margin-bottom: 10px;
+  color: white;
+  font-size: 20px;
+}
+
+.dance-header2 {
+  padding: 10px;
+}
+
+.dance-disabled {
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/css/dance-landing.css]: this too? not in haml

File: pegasus/sites.v3/code.org/public/dance-landing.haml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    %img.dance-img{src:"/shared/images/fit-150/hadi_partovi.jpg"}
+  %h1.dance-heading= I18n.t(:hoc2018_dance_create_your_own_dance_title)
+
+  .col-25.dance-tutorial.dance-blocks-tutorial
+    .tutorial-box
+      %h2.dance-h2= I18n.t(:hoc2018_dance_party_box_title)
+      .dance-tutorial-info
+        %p.dance-tutorial-description= I18n.t(:hoc2018_dance_party_box_description)
+        %p.dance-tutorial-specs= I18n.t(:hoc2018_dance_party_specs)
+      %a{:href => '/api/hour/begin/starwarsblocks', :target=>'_self'}
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/dance-landing.haml]: You should be able to hit this api with dance party so you can update here and other places in the file

**Comment by Erin007** [comment]
@nkiruka just saw your latest commit. Don't bother applying the feedback to the css file; address the bigger issue in the changes requested notes first.

**Comment by nkiruka** [comment]
@Erin007 Good ideas.  My initial approach is to build out the page and then refactor after (I already created a shared css file that will house the styles shared by craft, dance & starwars).

**Pull request closed without merging**